### PR TITLE
#7 added warning about URL to doc

### PR DESCRIPTION
**Pull request by apocarteres**

### CODE REVIEW

**Comment by dmarkov** [comment]
I'll find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by dmarkov** [comment]
@krzyk review this one please

File: src/main/java/com/jcabi/w3c/Validator.java [hunk 1]
```diff
@@ -65,6 +65,11 @@
  * from it, but it will be valid.
  *
  * <p>Objects of this interface should be immutable and thread-safe.
+
+ * <p>Please <b>don't use</b> URL as argument of
```
Comment (krzyk) [inline src/main/java/com/jcabi/w3c/Validator.java]: Please move this comment to `validate` method, and maybe change it slightly  to something like: This method  expects content of a document which should be validated, not URL to the document.

**Comment by apocarteres** [comment]
please check the updated version

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by dmarkov** [comment]
@krzyk **15 mins** added to your account (payment number `50663594`), many thanks for your contribution!; +15 to your rating, your total score is [+2716](http://www.netbout.com/b/35035?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy

**Pull request merged**